src/components/Dashboard/DashboardDevice.tsx

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
lets use ``// eslint-disable-next-line @typescript-eslint/camelcase`` instead

Reply:
well all device states are not using camelCase... so eslint complain about every single field, that's why it's disabled on file

Reply:
Yes, but it's better to use it precisely rather than completely disable